data.sql

```diff
@@ -162,4 +162,8 @@ FULL JOIN specializations ON specializations.vets_id = vets.id
 WHERE vets.name = 'Maisy Smith' 
 GROUP BY S 
 ORDER BY CNT 
-DESC LIMIT 1;
\ No newline at end of file
+DESC LIMIT 1;
+
+--Performance Improvement--
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/215529858-d2664673-3493-43ce-a646-3ce6890b2fb7.png)

**[Optional]**

- [ ] It would be nice if you add some most frequently asked questions about this application and their answers to make your readme file professional :+1: